imap_processing/cdf/config/imap_hi_variable_attrs.yaml

```diff
@@ -31,13 +31,13 @@ default_uint32_attrs: &default_uint32
   VALIDMAX: 4294967295
   dtype: uint32
 
-default_uint64_attrs: &default_uint64
+default_int64_attrs: &default_int64
   <<: *default
   FILLVAL: 18446744073709551615
   FORMAT: I20
   VALIDMIN: 0
   VALIDMAX: 18446744073709551615
```

Comment:
Are these still good with change from `uint64` to `int64`?

Reply:
Ah... Good catch! Nope, those are not correct.